contracts/LifCrowdsale.sol

```diff
@@ -346,18 +357,56 @@ contract LifCrowdsale is Ownable, Pausable {
       foundationWallet.transfer(foundationBalanceCapWei);
 
       // TODO: create the MVM with a start block that equals one month after crowdsale ends
-      LifMarketValidationMechanism newMVM = new LifMarketValidationMechanism(
+      MVM = new LifMarketValidationMechanism(
         address(token), block.timestamp.add(10), 30 days, MVMPeriods, foundationWallet
       );
-      newMVM.fund.value(mmFundBalance)();
-      newMVM.transferOwnership(foundationWallet);
 
-      MVM = address(newMVM);
+      mintExtraTokens(uint256(MVMPeriods));
+
+      MVM.fund.value(mmFundBalance)();
+      MVM.transferOwnership(foundationWallet);
 
     }
   }
 
   /**
+     @dev Internal. Distribute the rest of the tokens among founders and advisors
+   */
+  function mintExtraTokens(uint256 foundationMonthsStart) internal {
+    // calculate how much tokens will the founders, foundation and advisors will receive
+    uint256 foundersTokens = token.totalSupply().mul(128).div(1000);
+    uint256 foundationTokens = token.totalSupply().mul(50).div(1000);
+    uint256 advisorsTokens = token.totalSupply().mul(72).div(1000);
+
+    // create the vested payment schedule for the founders
+    foundersVestedPayment = new VestedPayment(
+      block.timestamp, 30 days, 48, 12, foundersTokens, token
+    );
+    token.mint(foundersVestedPayment, foundersTokens);
+    foundersVestedPayment.transferOwnership(foundationWallet);
+
+    // create the vested payment schedule for the foundation
+    uint256 foundationPaymentStart = foundationMonthsStart.mul(604800);
+    foundationVestedPayment = new VestedPayment(
+      block.timestamp.add(foundationPaymentStart), 30 days,
+      foundationMonthsStart, 0, foundationTokens, token
+    );
+    token.mint(foundationVestedPayment, foundationTokens);
+    foundationVestedPayment.transferOwnership(foundationWallet);
+
+    // transfer the token for advisors and future employees to the foundation
+    token.mint(foundationWallet, advisorsTokens);
+
+    // finish the minting of the token and unpause it
```

Comment:
maybe it's better to do the `finishMinting`, `unPause` and `transferOwnership` in `finalize`, after the call to `forwardFunds`, what do you think?

Reply:
we cant do the finishMinting before, we need to mint the extra tokens inside forwardFunds.

Reply:
I agree we can't do it before `forwardFunds`, that's why I'm saying to do it after the call to `forwardFunds` ;)